# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-04-11 13:01+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: account/views.py:73
msgid "Username and password cannot be empty"
msgstr "帳號及密碼不可為空"

#: account/views.py:81
msgid "User does not exist.Please register an account first"
msgstr "帳號不存在，請先註冊帳號"

#: account/views.py:96 account/views.py:512
msgid "Something went wrong. Please try again"
msgstr "錯誤，請在嘗試一次"

#: account/views.py:102 account/views.py:518
msgid "Username or password is not correct"
msgstr "帳號或密碼不正確"

#: account/views.py:113 account/views.py:534
msgid "You have successfully logged out"
msgstr "已成功登出"

#: account/views.py:145 account/views.py:364
msgid "Invalid email address"
msgstr "信箱格式不正確"

#: account/views.py:170
msgid ""
"User has successfully created. Please check email for account validation"
msgstr "已經成功建立使用者，請進行信箱驗證"

#: account/views.py:174
msgid ""
"User has successfully created. Email sent failed. Please resend the email to "
"validate the account"
msgstr "已經成功建立使用者，信箱傳送失敗，請再次傳送驗證信"

#: account/views.py:177
msgid "User name has existed"
msgstr "使用者名稱已存在"

#: account/views.py:196
msgid "Please register an account first"
msgstr "請先註冊帳號"

#: account/views.py:199 account/views.py:262
msgid "Account has been confirmed. No need to get confirmation email again"
msgstr "帳號已驗證，不需再次傳送驗證信"

#: account/views.py:211
msgid "Please wait for a while to resend the code"
msgstr "請稍候再嘗試"

#: account/views.py:229
msgid "Confirmation email has been sent"
msgstr "已寄送驗證信"

#: account/views.py:231
msgid "Failed to send email"
msgstr "傳送信件失敗"

#: account/views.py:256
msgid "Invalid code"
msgstr "認證失敗"

#: account/views.py:269
msgid "User name has been changed"
msgstr "帳號已更改"

#: account/views.py:273
msgid "Account has been validated"
msgstr "帳號已驗證"

#: account/views.py:276
msgid "Account validation failed"
msgstr "帳號驗證失敗"

#: account/views.py:294
msgid "Send reset email failed. Please send again"
msgstr "傳送信件失敗，請在傳送一次"

#: account/views.py:297
msgid ""
"New password has sent to your mailbox. Please check and use the new password "
"to login"
msgstr "新的密碼已寄送至信箱，請收信後使用新密碼來登入"

#: account/views.py:301
msgid "User does not exist"
msgstr "使用者不存在"

#: account/views.py:352 account/views.py:431 account/views.py:459
#: account/views.py:579 account/views.py:624 account/views.py:662
#: account/views.py:713 chatbot/views.py:179 chatbot/views.py:207
#: chatbot/views.py:237 chatbot/views.py:288 chatbot/views.py:301
#: chatbot/views.py:320 chatbot/views.py:330 chatbot/views.py:377
#: chatbot/views.py:390 chatbot/views.py:409 chatbot/views.py:419
#: faq/views.py:107 faq/views.py:179 faq/views.py:191 faq/views.py:213
#: faq/views.py:223 faq/views.py:284 faq/views.py:296 faq/views.py:319
#: faq/views.py:329 faq/views.py:468 history/views.py:57 paidtype/views.py:103
#: report/views.py:85 taskbot/views.py:88 taskbot/views.py:123
#: taskbot/views.py:153
msgid "Not found"
msgstr "資料不存在"

#: account/views.py:368
msgid "Username has existed.Please use other one"
msgstr "帳號名稱已存在，請輸入其他名稱"

#: account/views.py:377
msgid "Sending confirmation email failed"
msgstr "寄送認證信失敗"

#: account/views.py:385
msgid "Confirmation email has sent.Please check your mailbox and login again"
msgstr "認證信已寄送，請至信箱進行認證後登入"

#: account/views.py:395
msgid "Old password cannot be empty"
msgstr "舊密碼不能為空"

#: account/views.py:401 account/views.py:592
msgid "Old password is not correct"
msgstr "舊密碼不正確"

#: account/views.py:406 account/views.py:590
msgid "Password has updated.Please login again"
msgstr "密碼已更新，請重新登入"

#: account/views.py:420
msgid "Please select one language"
msgstr "請選擇語言"

#: account/views.py:423 account/views.py:616 chatbot/views.py:193
#: chatbot/views.py:336 chatbot/views.py:425 faq/views.py:227 faq/views.py:333
#: taskbot/views.py:110
msgid "Update succeeded"
msgstr "更新成功"

#: account/views.py:426 account/views.py:475 account/views.py:572
#: account/views.py:678 account/views.py:753 chatbot/views.py:168
#: chatbot/views.py:195 chatbot/views.py:250 chatbot/views.py:338
#: chatbot/views.py:427 faq/views.py:204 faq/views.py:229 faq/views.py:309
#: faq/views.py:335 faq/views.py:413 paidtype/views.py:90 taskbot/views.py:79
#: taskbot/views.py:112 taskbot/views.py:166
msgid "No content"
msgstr "無內容"

#: account/views.py:436 account/views.py:629 chatbot/views.py:210
#: taskbot/views.py:126
msgid "Please confirm the deletion first"
msgstr "請先確認是否刪除"

#: account/views.py:443 account/views.py:636
msgid "Deleting account failed"
msgstr "刪除帳號失敗"

#: account/views.py:465 account/views.py:668 chatbot/views.py:241
#: taskbot/views.py:157
msgid "Please enter the password"
msgstr "請輸入密碼"

#: account/views.py:471 account/views.py:674 chatbot/views.py:246
#: taskbot/views.py:162
msgid "Delete confirmed"
msgstr "確認刪除"

#: account/views.py:473 account/views.py:676 chatbot/views.py:248
#: taskbot/views.py:164
msgid "Password is not correct"
msgstr "密碼不正確"

#: account/views.py:495
msgid "User does not exist. Please contact admin."
msgstr "帳號不存在，請聯繫客服人員"

#: account/views.py:499
msgid "Only staff can login this page"
msgstr "必須為管理員帳號"

#: account/views.py:568
msgid "Create new agent succeeded"
msgstr "創建管理員帳號成功"

#: account/views.py:570 faq/views.py:202 faq/views.py:307 history/views.py:128
msgid "Create failed"
msgstr "創建失敗"

#: account/views.py:600
msgid "User name existed"
msgstr "使用者名稱已存在"

#: account/views.py:610
msgid "Do not support this language"
msgstr "不支援此語言"

#: account/views.py:621 account/views.py:658
msgid "Superuser cannot be deleted"
msgstr "不能刪除超級管理員帳號"

#: account/views.py:727
msgid "Invalid type"
msgstr "方案不正確"

#: account/views.py:749
msgid "Update successed"
msgstr "更新成功"

#: account/views.py:751
msgid "paid_type cannot be empty"
msgstr "方案不能為空"

#: chatbot/views.py:115
msgid "Reach bot create limitation"
msgstr "已到建立機器人數量上限"

#: chatbot/views.py:127
msgid "Reach faq create limitation"
msgstr "已到問答數量上限"

#: chatbot/views.py:158 taskbot/views.py:70
msgid "Create bot failed. Cause by NLU error."
msgstr "建立機器人失敗"

#: chatbot/views.py:165 taskbot/views.py:77
msgid "Create bot failed"
msgstr "建立機器人失敗"

#: chatbot/views.py:217 taskbot/views.py:134
msgid "Deleting bot failed"
msgstr "刪除機器人失敗"

#: chatbot/views.py:291 chatbot/views.py:304 chatbot/views.py:323
msgid "Not allowed to get user line data"
msgstr "不允許此操作"

#: chatbot/views.py:380 chatbot/views.py:393 chatbot/views.py:412
msgid "Not allowed to get user fb data"
msgstr "不允許此操作"

#: faq/views.py:117
msgid "Faq group exceeded upper limit"
msgstr "問答組已到數量上限"

#: faq/views.py:126
msgid "Create faq group failed"
msgstr "創建問答組失敗"

#: faq/views.py:363 faq/views.py:375
msgid "File type is not correct. Should be type utf8 or big5."
msgstr "格式不正確，需為utf8或big5"

#: faq/views.py:371
msgid "CSV column is not correct"
msgstr "CSV 欄位數量不正確"

#: faq/views.py:400
msgid "Group over limitation"
msgstr "問答組超過上限"

#: faq/views.py:408
msgid "Upload succeeded"
msgstr "上傳成功"

#: faq/views.py:411 faq/views.py:454
msgid "Bot not found"
msgstr "機器人不存在"

#: faq/views.py:474
msgid "Training bot succeeded"
msgstr "成功訓練機器人"

#: history/views.py:100
msgid "Chatbot not found"
msgstr "機器人不存在"

#: history/views.py:107
msgid "Matching result not found"
msgstr "推薦紀錄不存在"

#: history/views.py:114
msgid "FAQ group does not exisit"
msgstr "問答組不存在"

#: history/views.py:125
msgid "Create new question successed"
msgstr "成功建立新問題"

#: paidtype/views.py:65 paidtype/views.py:98
msgid "Not allowed"
msgstr "不允許此操作"

#: paidtype/views.py:80
msgid "Create paidtype failed"
msgstr "建立方案失敗"

#: taskbot/views.py:97
msgid "Key missing or empty: robot_name"
msgstr "缺少robot_name"

#~ msgid "English"
#~ msgstr "英文"

#~ msgid "Simplify Chineses"
#~ msgstr "簡體中文"

#~ msgid "Traditional Chineses"
#~ msgstr "繁體中文"
